src/sentry/models/event.py

```diff
@@ -1,11 +1,17 @@
+from __future__ import annotations
+
+from collections.abc import Iterator, Mapping, MutableMapping
+from typing import Self, TypeVar
+
 import orjson
 from sentry_relay.processing import StoreNormalizer
 
 from sentry.db.models import NodeData
-from sentry.utils.canonical import CanonicalKeyDict
 
+V = TypeVar("V")
```

Comment:
What does this corresponds to? Can you add a comment?

Reply:
I think every caller uses it as `Any` right now -- but it's the (conventional name for) value type used below